### PR TITLE
Remove delete

### PR DESCRIPTION
**Pull request by WilliamRyank**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -67,7 +67,7 @@ dependencies {
 }
 
 shadowJar {
-    archiveName = 'addressbook.jar'
+    archiveName = 'organice.jar'
```
Comment (TheRealRavager) [inline build.gradle]: Sneaky

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -308,20 +308,16 @@ Lists all patients only.
 Edits the information of the person with the specified NRIC to the attributes specified.
 It is up to the user to specify which attributes should be updated. +
 
-Format: `edit ic/**NRIC**`
```
Comment (TheRealRavager) [inline docs/UserGuide.adoc]: we changed it to not require `ic/` anymore?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -308,20 +308,16 @@ Lists all patients only.
...
+Format: `edit **NRIC**`
 
-Optional parameters: any parameter from the <<List of Attributes>>
-
-After editing, ORGANice will display a form to show what information will be stored about
-the person after confirming the changes. The user is free to edit the fields before
-changes are confirmed.
+Currently in this version, you can only edit the `Phone` and `Name` attributes.lew
```
Comment (TheRealRavager) [inline docs/UserGuide.adoc]: `attributes.lew`?

File: docs/UserGuide.adoc (hunk 2 above)
Comment (C-likethis123) [reply docs/UserGuide.adoc]: Yes

**Pull request merged**